src/org/labkey/test/components/ui/entities/EntityInsertPanel.java

```diff
@@ -234,36 +249,56 @@ protected boolean isVisible(Locator locator)
         }
     }
 
```

Comment:
My investigation of switching panes between grid/file upload shows that the switch doesn't show/hide DOM elements, it  destroys them but the DOM flutter occurs within the ComponentElement so it doesn't cause the componentElement to go stale.  Interestingly, state is well-preserved between panes (if you've attached a file to the fileUpload panel and switch to grid and then come back, the file remains attached and the preview is still there).  Invalidating the cache here seems to be a best defensive practice